package-examples/wordpress/Kptfile

```diff
@@ -1,39 +1,10 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
```

Comment:
We're going to use the as the example in Chapter 3 of the book, so let's have it encompass all the best practices.

Maybe include `info` fields:

https://github.com/GoogleContainerTools/kpt/blob/next/pkg/api/kptfile/v1alpha2/types.go

Reply:
Added info fields but kubeval is not working like in yesterday's demo. We can add it later.